components/insight/build/build.xml

```diff
@@ -123,13 +123,12 @@
 	<condition property="java.not.ok" value="true">
 	  <not>
 	    <or>
-	      <contains string="${ant.java.version}" substring="1.6"/>
 	      <contains string="${ant.java.version}" substring="1.7"/>
 	      <contains string="${ant.java.version}" substring="1.8"/>
 	    </or>
 	  </not>
 	</condition>
-	<fail if="java.not.ok" message="Must use JDK 1.6.x or higher to build."/>
+	<fail if="java.not.ok" message="Must use JDK 1.7.x or higher to build."/>
```

Comment:
https://ci.openmicroscopy.org/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/51/console failed with this change. Need to investigate. Marked the PR as breaking to let the builds pass.